src/app/gigs/contract/contract-overview.tsx

```diff
@@ -232,11 +157,33 @@ export const ContractOverview = ({
                 projectId={contract.id}
                 isClient={isClient}
                 milestoneId={milestone.id}
+                handleMilestoneFunding={() =>
```

Comment:
Consider following standard event handler naming convention. [More info](https://jaketrent.com/post/naming-event-handlers-react/).
```suggestion
                onMilestoneFunded={() =>
```
or something similar.